docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -104,6 +104,213 @@ adapter.isValid(dayjs('2022-04-17T15:30'));
 If you are just passing an adapter to `LocalizationProvider`, then you can safely skip this section.
 :::
 
+### Remove the `getDiff` method
```

Comment:
I've been a lot more exhaustive that needed and I hope nobody will complain about the removal of these methods :laughing: